merge_rgba/__init__.py

```diff
@@ -3,7 +3,17 @@
 import numpy as np
 import rasterio
 from rasterio.transform import Affine
-from rasterio._base import get_index  # get_window
+
+try:
+    from rasterio.transform import rowcol  # rasterio>=1.0
+except ImportError:
+    # Use wrapper function around deprecated rasterio pre-1.0 function
+    from rasterio._base import get_index
+    from rasterio.transform import guard_transform
+
+    def rowcol(transform, x, y, op, precision):
+        return get_index(
+            x, y, guard_transform(transform), op=op, precision=precision)
```

Comment:
@sgillies This was the only way I could come up with to keep compatibility between rasterio versions. `_base.get_index` went away and has a slightly different function signature requiring a wrapper function. Ideas?